### PR TITLE
[Docs] Investigate the possibility to add a plugin...

### PR DESCRIPTION
**Pull request by dzhigzhitzhapov**
Add a glossary draft with the ezglossary plugin.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ nav:
         - Docker: capi-docker.md
         - vSphere: capi-vsphere.md
   - FAQ: faq.md
+  - Terminology: terminology.md
```
Comment (makhov) [inline mkdocs.yml:39]: Maybe, put it in the Reference section?

**Comment by makhov** [comment]
A couple of screenshots

image
image

**Review by makhov — APPROVED** [review]

File: docs/status.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ One of the directions we're looking at is the have k0smotron working as a Cluste
...
 
 Some of the areas we know have lot of shortcomings currently:
+
 - Control Plane configurability: As you know, k0s itself has lot of configurability, we plan to enable full configurability of the k0s control plane
+
 - Control plane exposing: Currently k0smotron only supports `NodePort` and `LoadBalancer` type services. While that is in itself quite ok quite often there's need to further configure e.g. annotations etc. on the created service to make them play nice with cloud provider implementations.
-- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
-- 
+
+- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
```
Comment (ologvinova) [inline docs/status.md:22]: is able to update -> can update

Comment (ologvinova) [inline docs/status.md:22]: controlplane -> control plane

comma after 'easily'

File: docs/terminology.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Terminology
+
+term:Control Plane
```
Comment (ologvinova) [inline docs/terminology.md:3]: plane (sentence case)

File: docs/status.md (hunk 2 above)
Comment (ologvinova) [inline docs/status.md:22]: While -> while (the same applies to the items above)

File: mkdocs.yml [hunk 4]
```diff
@@ -64,6 +67,10 @@ extra_css:
   - stylesheets/extra.css
 plugins:
   - search
+  - ezglossary:
+        section_config:
```
Comment (ologvinova) [inline mkdocs.yml:71]: do we require 4 indentations here? or 2 is enough?

File: docs/terminology.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Terminology
```
Comment (ogusarenko) [inline docs/terminology.md:1]: please add the syntax for referring to a term here as a comment

File: docs/status.md (hunk 2 above)
Comment (dzhigzhitzhapov) [reply docs/status.md:22]: these one will be corrected in Kory's commit

Comment (dzhigzhitzhapov) [reply docs/status.md:22]: Goes to Kory :)

File: mkdocs.yml (hunk 4 above)
Comment (dzhigzhitzhapov) [reply mkdocs.yml:71]: I think we don't. It works fine with two indentations locally.

File: docs/terminology.md (hunk 3 above)
Comment (ologvinova) [reply docs/terminology.md]: not resolved yet

File: docs/status.md [hunk 6]
```diff
@@ -14,9 +14,7 @@ One of the directions we're looking at is the have k0smotron working as a Cluste
...
 
 Some of the areas we know have lot of shortcomings currently:
-
 - Control Plane configurability: As you know, k0s itself has lot of configurability, we plan to enable full configurability of the k0s control plane
-
 - Control plane exposing: Currently k0smotron only supports `NodePort` and `LoadBalancer` type services. While that is in itself quite ok quite often there's need to further configure e.g. annotations etc. on the created service to make them play nice with cloud provider implementations.
-
-- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
\ No newline at end of file
+- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
```
Comment (ologvinova) [inline docs/status.md:19]: Can we rebase this part on top of https://github.com/k0sproject/k0smotron/pull/485 that was merged yesterday?

**Review by ologvinova — APPROVED** [review]

**Pull request merged**